docs/UserGuide.md

```diff
@@ -58,105 +102,154 @@ management tasks done faster than traditional GUI apps.
 
 </div>
 
-### Adding an item: `add`
+### 3.1. Viewing help : `help`
 
-Adds an item to the inventory.
+If you are unsure about the commands and want to seek help, this command opens up this User Guide on your browser 
+**if you are connected to the internet**, where you can get detailed information on how to use each feature in 
+StoreMando. If you are not connected to the internet or have no browser installed on the device you are using to access 
+StoreMando, this command would then open up a pop-up help window which would provide you with the link to this 
+User Guide.
+
+Format: `help`
+
+### 3.2. Adding an item : `add`
+
+Have an item to add to the inventory? Then this command is the right one for you!
 
 Format: `add n/ITEM_NAME l/LOCATION q/QUANTITY [e/EXPIRY_DATE] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 An item can have any number of tags (including 0)
-
 </div>
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 Expiry date of an item is optional.
-
 </div>
 
 Examples:
+* `add n/Chocolate Milk l/kitchen refrigerator q/2` adds 2 Chocolate Milk to the kitchen refrigerator. 
+* `add n/Sunshine Bread l/bedroom q/10 e/2020-01-01 t/favourite` adds 10 Sunshine Bread with expiry date, 2020-01-01, 
+  and "favourite" tag to the bedroom.
 
-* `add n/Chocolate Milk l/Kitchen Refrigerator q/2`
-* `add n/Sunshine Bread l/Bedroom q/10 e/2020-01-01 t/favourite`
-
-### Listing all items : `list`
-
-Shows a list of all items in the inventory.
+### 3.3. Editing an item : `edit`
 
-Format: `list` `list [l/LOCATION]` `list [t/TAG]`
+If you want to edit any details of an existing item in the inventory, you can do so by using this command. You don't 
+have to worry about adding any wrong information as you can edit them any time!
 
-* You can view all items in the inventory by typing 'list' without specifying location/tag.
-* The search is case-insensitive. e.g 'room' will match 'Room'.
-* The order of keywords for location search does not matter. e.g. 'Room Living' will match 'Living Room'.
-* Tag keyword only can contain one single word.
-* Only full words will be matched e.g. 'Room' will not match 'Bedroom'.
-* Only location/tag can be searched.
-* Location matching uses each word in the String to do 'AND' search e.g. 'Room' will match 'Living room' but 'Living
-  room 1' will not match 'Living room'
-
-### Editing an item's details : `edit`
+Format: `edit INDEX [n/ITEM_NAME] [l/LOCATION] [q/QUANTITY] [e/EXPIRY_DATE] [t/TAG]…​`
 
-Editing an existing item in the inventory.
+<div markdown="block" class="alert alert-info">
 
-Format: `edit INDEX [n/ITEM_NAME] [l/LOCATION] [q/QUANTITY] [e/EXPIRY_DATE] [t/TAG]…​`
+**:information_source: Notes about the edit command:**<br>
 
 * Edits the item at the specified `INDEX`. The index refers to the index number shown in the displayed item list. The
   index **must be a positive integer** 1, 2, 3, …​
```

Comment:
I think this info about the index being positive is not required because it has been stated very clearly in the sentence before this that 'The index refers to the index number shown in the displayed item list.' :)

Reply:
I think it is better to be as explicit as possible in the user guide and there is no harm in doing so.